__tests__/endpoints.test.js

```diff
@@ -0,0 +1,96 @@
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data/index");
+const request = require("supertest");
+const app = require("../API/app");
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  db.end();
+});
+
+describe("01 GET /api/topics", () => {
+  describe("Functionality", () => {
+    test("When correctly called, endpoint should return a 200 status code", () => {
+      return request(app).get("/api/topics").expect(200);
+    });
+
+    test("When called, endpoint should return a body containing an array of topic Objects", () => {
+      return request(app)
+        .get("/api/topics")
+        .then(({ body }) => {
+          const { topics } = body;
+          expect(topics.constructor).toEqual(Array);
+          topics.forEach((topic) => {
```

Comment:
What happens with a for each and an empty array? With an empty array a forEach will skip the iteration, if you have an expect statement inside that loop, the expect is skipped and the test passes incorrectly. While this test on line 21 has an expect outside of the loop, the test on line 33 does not. You should be expecting an array of a length greater than 0. As we're on test data technically then you say the length is 3, because it is.